package/src/lsif.ts

```diff
@@ -115,11 +120,7 @@ export const mkIsLSIFAvailable = () => {
             if (!response.ok) {
                 return false
             }
-            const available: boolean = await response.json()
-            if (available) {
-                console.log('Using LSIF data', { repository, commit, file })
```

Comment:
Why remove this indicator?

Reply:
It was no longer symmetric with GraphQL, which didn't have a way to determine if there was a dump without actually making the query first.

After changes to RFC 74, this will be reworked again.

Reply:
I added logging for each request instead.